### DISCUSSION TITLE
Show the bounds of a model with dimensions in the scene

### DISCUSSION THREAD

**Discussion by buddy90210** [google/model-viewer#2225] [Q&A]
How to show the bounds of a model with dimensions in the scene? Is it possible?
![1_ssY-tVnMUuxFFOd78m2ekw](https://user-images.githubusercontent.com/12248799/94826989-98887680-0410-11eb-953a-5eb6657491f2.gif)

**Comment by elalish** [comment]
hmm, not very easily. You could add the box to your glTF, and potentially add the measurement labels as hotspots. If you'd like to be able to turn the box on and off, you'll need more scene-graph functionality than we currently have, but that is on our roadmap. Of course this would only be available in 3D and WebXR modes. SceneViewer and QuickLook don't expose this kind of functionality.

**Reply by blippyblips** [reply]
"You'll need more scene-graph functionality than we currently have, but that is on our roadmap."

How is the possibility of displaying bounding boxes or boxes in general looking now? Figured it's been a bit over a year so maybe some of this has already been made?

**Reply by uprealdigital** [reply]
Would love that feature as well!
